README.md

```diff
@@ -3,14 +3,18 @@
 This is a collection of games that may help you and your friends kill the boredom of COVID19 quarantine.
 
 
-## Development 
+## Development
 Install dependencies `npm install`.
 
 Run react dev server with `npm start`.
 
 Run game server with: `node -r esm src/server.js`.
 
-Setting `debug=true` in URL query will show debug overlay. 
+Setting `debug=true` in URL query will show debug overlay.
+
+To automatically improve the code quality you can use [pre-commits](https://pre-commit.com):
+1. `brew install pre-commit`
+2. `pre-commit install`
```

Comment:
Or we can add some npm library that takes care of it https://github.com/typicode/husky

Reply:
+1 for husyky
it's a standard in JS projects